src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+
+  margin-top: 8px;
+  margin-bottom: 8px;
+  gap: 17px;
```

Comment:
You're using the `gap` property which is not fully supported by all browsers. Although it's a convenient shorthand for `row-gap` and `column-gap`, it might be safer to use these two properties instead to ensure compatibility across all browsers.